prettier.config.js

```diff
@@ -0,0 +1,33 @@
+/** @type {import('prettier').Config} */
```

Comment:
Can you remove, I think these setups tend to be quite subjective preference, so would want to leave that up to the individual user.